commit ff5178ae73b2d73d88a778919e3d6a24a22ada84
Author: bowrocker <jon.anderson@joyent.com>
Date:   2019-07-25T09:37:11-04:00 (2 months ago)
    
    MANATEE-415 manatee-sitter spawns excessive zfs recv processes when manatee-backupserver isn't running on upstream peer

diff --git a/package.json b/package.json
index c9ab264..16a1323 100644
--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     },
     "dependencies": {
         "manta": "1.2.6",
-        "manatee": "git+https://github.com/joyent/manatee.git#6e802583f517e74f8e6c1ccab26b01ce2bbbf8f9",
+        "manatee": "git+https://github.com/joyent/manatee.git#492303bde87c4cc8b0c941b2bac117b2fae86fb9",
         "carrier": "0.2.0"
     },
     "sdcDependencies": {
